internal/databases/sqlserver/backup_push_handler.go

```diff
@@ -14,6 +15,9 @@ import (
 	"github.com/wal-g/wal-g/utility"
 )
 
+const DatabaseBackupItem = "DATABASE"
+const LogBackupItem = "LOG"
```

Comment:
Nope.  DATABASE and LOG backup are completely different operation.
SQL queires are just similar now by will diverge in the future.

Reply:
fixed in b5ae4f4